rust/frontend/src/server.rs

```diff
@@ -863,11 +863,13 @@ async fn create_collection(
         "op:create_collection",
         format!("tenant:{}", tenant).as_str(),
     ]);
-    let index_type = DistributedIndexTypeParam::try_from(&payload.metadata)?;
-    // spann index not allowed.
-    if index_type.index_type == DistributedIndexType::Spann && !server.config.enable_span_indexing {
-        return Err(ValidationError::SpannNotImplemented)?;
+
+    if let Some(config) = payload.configuration.as_ref() {
+        if config.spann.is_some() {
```

Comment:
This does not look correct. Spann feature is gated by flag `server.config.enable_span_indexing` so if that is set to false only then we should prohibit user from creating a spann index otherwise we should allow

Reply:
whoops, yes--I think this got lost in a rebase at some point
will fix